invenio_search/walkers/elasticsearch.py

```diff
@@ -123,6 +120,11 @@ def _f(keyword):
     @visitor(DoubleQuotedValue)
     def visit(self, node):
         def _f(keyword):
+            if keyword == [u'exactauthor.raw']:
```

Comment:
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/config.py#L293-L297

Here it is